compositional_skills/writing/grounded/objection_handling/attribution.txt

```diff
@@ -0,0 +1,6 @@
+﻿Title of work: Objection handling skills
+Link to work: https://docs.google.com/document/d/1XZjABlQsE1L7-AYdXOb_lwBNEg9lKOrVuRb_HjsGY-o/edit?usp=sharing 

```

Comment:
Also, I'm not 100% sure if the legal team can allow content from random doc.google.com domains.